relayer/create_ibc_channel.go

```diff
@@ -69,64 +112,49 @@ func (r *Relayer) CreateIBCChannel(override bool, logFileOption utils.CommandOpt
 	}, nil
 }
 
+// waitForValidRollappHeight waits for the rollapp height to be greater than 2 otherwise
```

Comment:
`Waiting for the rollapp height on the hub to be greater than initial height` not needed anymore?

Reply:
no. because we open a connection optimistically so only the rollapp height is important.